packages/terra-framework-docs/src/terra-dev-site/doc/pills/example/AppliedFiltersToolbar.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import classNames from 'classnames/bind';
+import FilterPills from 'terra-pills';
+import { Placeholder } from '@cerner/terra-docs';
+
+import styles from './FilterPillCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const AppliedFiltersToolbar = () => {
+  const pillsData = [];
+  for (let pillCount = 1; pillCount < 11; pillCount += 1) {
+    const pillData = {
+      label: `Selection ${pillCount}`,
```

Comment:
That would probably add value to the readability of docs..!! 

But I think translating examples will require involvement of Translations team which is now only limited for translating client viewable text. 

Since Doc-site are mostly referred by Developers I am not sure if we require translations for developer guides with cost of Translating the strings on Doc site.